core/libs/models/src/crypto.rs

```diff
@@ -8,7 +8,7 @@ use uuid::Uuid;
 pub type AESKey = [u8; 32];
 pub type DecryptedDocument = Vec<u8>;
 pub type EncryptedDocument = AESEncrypted<DecryptedDocument>;
-pub type EncryptedUserAccessKey = RSAEncrypted<AESKey>;
+pub type EncryptedUserAccessKey = AESEncrypted<AESKey>;
```

Comment:
what's the idea for this change?

Reply:
With ECDH 2 parties knowing each others public key can generate a shared secret without talking to one another. The concept of encrypting something with Elliptic Curve is to encrypt it with this shared secret.

Reply:
shouldn't it be encrypted with ECDH instead of AES then?

Reply:
The shared secret is an arbitrary `u8; 32` we use it with AES as a key to encrypt. When we do this the resulting value is AESEncrypted